src/languages/en.ts

```diff
@@ -567,7 +567,7 @@ export default {
         settledElsewhere: 'Paid elsewhere',
         settleExpensify: ({formattedAmount}: SettleExpensifyCardParams) => (formattedAmount ? `Pay ${formattedAmount} with Expensify` : `Pay with Expensify`),
         payElsewhere: 'Pay elsewhere',
-        nextSteps: 'Next Steps',
+        nextStep: 'Next Steps',
```

Comment:
Should it be renamed if it's corresponding to the text it means?

Reply:
Yeah, good question. Let me ask @mountiny about it.

Reply:
I think this is good now, the design has the plural name in it but in sake of consistency elsewhere I think its better ot keep the name singular here